### PR TITLE
(feature) Better formatting and sanitization for vacancy text

### PR DESCRIPTION
**Pull request by despo**
Allows
 `p`, `br`, `strong`, `em`, `ul`, `li`, `h1`, `h2`, `h3`, `h4`, `h5` tags 

- sanitizes scraped vacancy's `job_description`
- sanitizes job_description, job_title, experience, education, qualifications and benefits on vacancy model
- render using `simple_format` to process new lines

Example scraped vacancy  
https://schooljobs.education.gov.uk/vacancies/deputy-headteacher-required-for-1st-september-2018 i

screen shot 2018-04-18 at 13 32 39 1

### CODE REVIEW

**Comment by despo** [comment]
[markdown? paragraph stuff?](https://trello.com/c/bHCzUC3C/71-markdown-paragraph-stuff)

File: spec/models/vacancy_spec.rb [hunk 1]
```diff
@@ -250,4 +250,54 @@
       end
     end
   end
+
+  context 'Content sanitization' do
+    it '#job_description' do
```
Comment (tahb) [inline spec/models/vacancy_spec.rb]: There's potential to put these similar methods into a shared example to DRY this up a bit.

**Comment by despo** [comment]
@tahb that's a css related issue.

**Comment by tahb** [comment]
Ah, yes! `<h1 class="heading-large">Header</h1>` equals:

screen shot 2018-04-19 at 11 33 47

**Comment by tahb** [comment]
Rebasing

**Pull request merged**